docs/api/features/pcoords.rst

```diff
@@ -78,11 +80,96 @@ Parallel coordinates can take a long time to draw since each instance is represe
+    from sklearn.datasets import load_iris
+    from yellowbrick.features import ParallelCoordinates
+
+    data = load_iris()
+
+    _, axes = plt.subplots(nrows=2, figsize=(9,9))
+
+    for idx, fast in enumerate((False, True)):
+        title = "Fast Parallel Coordinates" if fast else "Standard Parallel Coordinates"
+        oz = ParallelCoordinates(ax=axes[idx], fast=fast, title=title)
+        oz.fit_transform(data.data, data.target)
+        oz.finalize()
+        oz.ax.grid(False)
+
+    plt.tight_layout()
 
 As you can see the "fast" drawing algorithm does not have the same build up of color density where instances of the same class intersect. Because there is only one line per class, there is only a darkening effect between classes. This can lead to a different interpretation of the plot, though it still may be effective for analytical purposes, particularly when you're plotting a lot of data. Needless to say, the performance benefits are dramatic:
 
-.. image:: images/fast_parallel_coordinates_speedup.png
+.. plot::
+    :include-source: False
+
+    import time
+    import matplotlib.pyplot as plt
+    from sklearn.datasets import load_iris
+    from yellowbrick.features import ParallelCoordinates
+    import pandas as pd
+
+    trials = 5
+    factors = np.arange(1, 11)
+
+    def pcoords_time(X, y, fast=True):
+        _, ax = plt.subplots()
+        oz = ParallelCoordinates(fast=fast, ax=ax)
+
+        start = time.time()
+        oz.fit_transform(X, y)
+        delta = time.time() - start
+
+        plt.cla()        # clear current axis
+        plt.clf()        # clear current figure
+        plt.close("all") # close all existing plots
+
+        return delta
+
+    def pcoords_speedup(X, y):
+        fast_time = pcoords_time(X, y, fast=True)
+        slow_time = pcoords_time(X, y, fast=False)
+
+        return slow_time / fast_time
+
+    data = load_iris()
+
+    speedups = []
+    variance = []
+
+    for factor in factors:
+        X = np.repeat(data.data, factor, axis=0)
+        y = np.repeat(data.target, factor, axis=0)
+
+        local_speedups = []
+        for trial in range(trials):
+            local_speedups.append(pcoords_speedup(X, y))
+
+        local_speedups = np.array(local_speedups)
+        speedups.append(local_speedups.mean())
+        variance.append(local_speedups.std())
+
+    speedups = np.array(speedups)
+    variance = np.array(variance)
+
+    series = pd.Series(speedups, index=factors)
+    _, ax = plt.subplots(figsize=(9,6))
+    series.plot(ax=ax, marker='o', label="speedup factor", color='b')
+
+    # Plot one standard deviation above and below the mean
+    ax.fill_between(
+        factors, speedups - variance, speedups + variance, alpha=0.25,
+        color='b',
+    )
+
+    ax.set_ylabel("speedup factor")
+    ax.set_xlabel("dataset size (number of repeats in Iris dataset)")
+    ax.set_title("Speed Improvement of Fast Parallel Coordinates")
+    ax.grid(True)
+    ax.margins(0)
```

Comment:
So I struggled with this one - this is actually benchmark code and was part of what @thekylesaurus contributed in #420. Although it could be something we want to rerun when we build the documentation, I would rather continue to report his original results. Thank you so much for your hard work on it, and I'm sorry that I forgot this was here and didn't get a chance to warn you. But perhaps it would be better if we put this code back into a `pcoord_benchmark.py` file and kept `.. image:: images/fast_parallel_coordinates_speedup.png` here?

Reply:
As requrested, I moved the code to pcoord_benchmark.py then dug through my git commits to find/restore fast_parallel_coordinates_speedup.png